website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -164,6 +172,26 @@ curl \
 }
 ```
 
+## Upload State and JSON State
+
+-> **Note:** While you can upload state version content in the same request when creating a state version, we strongly recommend that you upload content separately.
```

Comment:
We try to minimize the number of callouts in the docs, if possible. Does this need to be a call out? It seems like this would get enough attention from being the first sentence:
```suggestion
You can upload state version content in the same request when creating a state version. However, we _strongly_ recommend that you upload content separately.
```
Also, why do we recommend this? Can we give the users more context?